build.cmd

```diff
@@ -478,12 +478,11 @@ if '%BUILD_PROTO_WITH_CORECLR_LKG%' == '0' (
 rem Build Proto
 if '%BUILD_PROTO%' == '1' (
   if '%BUILD_PROTO_WITH_CORECLR_LKG%' == '1' (
-
-    pushd .\lkg & %_dotnetexe% restore &popd
-    @if ERRORLEVEL 1 echo Error: dotnet restore failed  && goto :failure
-
-    pushd .\lkg & %_dotnetexe% publish project.json -o %~dp0\Tools\lkg -r win7-x64 &popd
-    @if ERRORLEVEL 1 echo Error: dotnet publish failed  && goto :failure
+echo on
```

Comment:
Why is `echo` turned on here?  The console output of the following commands will still be visible, just not what they were invoked with.

Otherwise, indention.